lib/rgit/cli.rb

```diff
@@ -292,8 +292,12 @@ def test(*args)
     end
 
     desc "log", "Show commit logs"
+    method_option :skip,
+      { type: :number,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.